### DISCUSSION TITLE
My Crypto asset from Fintoch@Defi wallet getting stuck (NODE CASH VOLUME)

### DISCUSSION THREAD

**Discussion by hamican** [community/community#59396] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Body

I need too know how should I do to get my Crypton asset could withdraw or transfer to Another wallet As before... After Fintoch launch fin-chain All my Crypton asset has been Change to Node Voucher and stuck in Fintoch wallet... 
What should I do?? I need Urgent feedback@advice
If I could withdraw all my asset, I will be thanks full

**Comment by queenofcorgis** [comment]
Hi @hamican ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Crypton/Finotech. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
